scripts/userdb/mysqlsetup.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
```

Comment:
Don't assume the user has bash on their machine.  You can use the `politeiad/scripts` as an example of how scripts should be written.

Reply:
Oh, I copied that from the cockroach tables setup script, will update all scripts to match politeiad' ones.